.gitignore

```diff
@@ -8,3 +8,4 @@ tmp-*
 .eslintcache
 .ncu
 package-lock.json
+.idea/
```

Comment:
I'm not blocking on this but this seems unrelated and perhaps should go in the user's own `.gitignore`?